nemo/export/quantize/quantizer.py

```diff
@@ -72,89 +79,13 @@ class Quantizer:
     for TensorRT-LLM deployment. This is useful to getting baseline results for a full-precision model.
     """
 
-    def __init__(
```

Comment:
Move quantization and inference arguments to `quantize()` and trtllm export related arguments to `export` so depending on what function a user wants to use, they can use it without worrying about other one. Also duplicated `decoder_type` to quantization parameters since it is used there as well

Reply:
OK. I only wonder if we can avoid specifying decoder type. Is there a use case when only `quantization_config` is used, but **not** `export_config`? If `quantization_config` always comes with `export_config`, we may resign from duplicating `decoder_type`

Reply:
Its possible that users just quantize in one script and separately export. Or they may skip export all together (currently int4 export is not working so we don't export in qat example). Also given that quantization post processing needs decoder type, quantization config should explicitly contain this parameter and not rely on some other config